app-office/calcurse/calcurse-4.6.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{6..8} )
+
+inherit autotools python-single-r1
+
+DESCRIPTION="a text-based calendar and scheduling application"
+HOMEPAGE="https://calcurse.org/"
+SRC_URI="https://calcurse.org/files/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep 'dev-python/httplib2[${PYTHON_MULTI_USEDEP}]')
+	sys-libs/ncurses:0="
+
+DEPEND="
+	${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.6.0-tinfo.patch
+)
+
+# Most tests fail.
+RESTRICT="test"
```

Comment:
Any chance you'd try to fix these failing tests?
https://dev.gentoo.org/~mgorny/python-guide/test.html there are many pointers to fixing test issues here.